dts/bindings/mtd/jedec,spi-nor.yaml

```diff
@@ -24,3 +24,28 @@ properties:
     type: phandle-array
     required: false
     description: RESETn pin
+  sfdp-basic-8:
+    type: int
+    required: false
+    default: 0xD810200C
+    description: |
+      Corresponds to the 8th DWORD of the JEDEC Basic Flash Parameter Table,
+      providing erase type 1 and 2 size and instruction.
+  sfdp-basic-9:
+    type: int
+    required: false
+    default: 0x520F
+    description: |
+      Corresponds to the 9th DWORD of the JEDEC Basic Flash Parameter Table,
+      providing erase type 3 and 4 size and instruction.
+  sfdp-basic-11:
+    type: int
+    required: false
+    default: 0x80
+    description: |
+      Corresponds to the 11th DWORD of the JEDEC Basic Flash Parameter Table,
+      providing page size.
```

Comment:
This should state that only bits 7:4 for page size are interpreted by the driver, at least for now, so people don't expect to be able to set program times.